config/services.yml

```diff
@@ -87,6 +87,7 @@ https://cmr.earthdata.nasa.gov:
         - application/x-netcdf4
     steps:
       - image: !Env ${QUERY_CMR_IMAGE}
+        is_sequential: true
```

Comment:
Add example and description of `is_sequential` to the adapting services user guide documentation.

Reply:
This will be addressed in HARMONY-1659